client/src/components/UsersView.js

```diff
@@ -22,17 +22,17 @@ export default function UsersView() {
       headers: { "Content-Type": "application/json" },
       body: JSON.stringify(inputBody),
     })
-      .then((res) => res.json())
-      .then(() => getUsers())
-      .catch((e) => console.error(e.stack));
+      .then( (res) => res.json() )
+      .then( () => getUsers() )
+      .catch( (e) => console.error(e.stack) );
   };
 
   // TODO: implement as a button
   const deleteUser = async (id) => {
-    fetch(`http://localhost:5000/users/${id}`, { method: "DELETE" })
-      .then(() => setUsers(users.filter((user) => user.uid !== id)))
-      .then(() => getUsers())
-      .catch((e) => console.error(e.stack));
+    fetch( `http://localhost:5000/users/${id}`, { method: "DELETE" } )
+      .then( () => setUsers( users.filter( (user) => user.uid !== id ) ) )

```

Comment:
It might look a little weird here when there's a lot, but at the same time, definitely helps make it clear, without the use of something like rainbow-brackets extension.